repo_policy_compliance/blueprint.py

```diff
@@ -51,6 +51,7 @@
 # Bandit thinks this is the token value when it is the name of the environment variable with the
 # token value
 CHARM_TOKEN_ENV_NAME = "CHARM_TOKEN"  # nosec
+DISALLOW_FORK_ENV_NAME = "DISALLOW_FORK"  # nosec
```

Comment:
Should we use the endpoint to configure the policy document?

Reply:
Due to the constraints of the flask application, it is decided to use environment variables for now.